### ISSUE TITLE
Don't assume bibids are unique

### ISSUE THREAD

**Issue by escowles** [pulibrary/cicognara-rails#233]
One MARC record in Voyager can be linked to multiple DCLNums, so we shouldn't use them as the local system identifier when importing from Plum.

There may be other places where we assume a 1-to-1 mapping between MARC records and DCLNums.

**Comment by jrgriffiniii** [comment]
This seems to be primarily localized in https://github.com/pulibrary/cicognara-rails/blob/master/app/models/marc_indexer.rb#L74

**Comment by jrgriffiniii** [comment]
Referencing https://bibdata.princeton.edu/bibliographic/368314.json as an example record (presumably the source for https://cicognara.org/catalog/245), the ARK does appear to be present in the source for extraction as a unique identifier.

**Comment by jrgriffiniii** [comment]
https://bibdata.princeton.edu/bibliographic/8524680 does not contain an ARK and is a record which is featured as seed data.

**Comment by escowles** [comment]
Yes, I think the DCL number should be unique for each contributor.  But since both Figgy and the Getty will have data from multiple contributors, we would still need to figure out which contributor an item was from.  We'll see how the contributor is listed in the Getty data, but in Figgy, the presence of the Vatican rights statement is what we've typically used to differentiate Princeton/Vatican records.
